_data/projects/traveler

```diff
@@ -0,0 +1,10 @@
+
+name: Traveler
+desc: Small Travelling app that shows the travel destinations in Sri Lanka
+site: https://github.com/SahanMonaara
```

Comment:
@SahanMonaara This should be your project repo link which is "https://github.com/SahanMonaara/Traveler/tree/master"